openstack_controller/tests/test_simple_api.py

```diff
@@ -306,11 +355,13 @@ def get_os_hypervisors_detail_post_v2_33_response(url, headers, params=None, tim
                 "rel": "next"
             }
         ]
-    }""")  # noqa: E501
+    }"""  # noqa: E501
```

Comment:
Could we get rid of all `# noqa: E501` now?

Reply:
Can't, it fails line length